components/Footer.tsx

```diff
@@ -1,43 +1,137 @@
 import NextLink from 'next/link'
+import Image from 'next/image'
 import Box from '@mui/material/Box'
 import Link from '@mui/material/Link'
 import Typography from '@mui/material/Typography'
-import { CKB_EXPLORER_URL, NERVINA_GITHUB_URL, NERVOS_URL } from 'utils'
+import Divider from '@mui/material/Divider'
 import Button from '@mui/material/Button'
+import Stack from '@mui/material/Stack'
+import { CKB_EXPLORER_URL, EXPLORER_TITLE, IMG_URL, NERVINA_GITHUB_URL, NERVOS_URL } from 'utils'
+import { Fragment } from 'react'
 
 const links = [
   { label: 'Nervos', url: NERVOS_URL },
   { label: 'GitHub', url: NERVINA_GITHUB_URL },
   { label: 'CKB Explorer', url: CKB_EXPLORER_URL },
 ]
-const Footer = () => (
-  <Box
-    component="footer"
-    display="flex"
-    flexDirection="column"
-    justifyContent="space-evenly"
-    sx={{ bgcolor: 'primary.dark', height: 'var(--footer-height)', color: 'white' }}
-  >
-    <Box component="div" textAlign="center">
-      {links.map(link => (
-        <NextLink href={link.url} key={link.label}>
-          <Button
-            href={link.url}
-            title={link.label}
-            color="inherit"
-            component={Link}
-            sx={{ textTransform: 'none' }}
-            target="_blank"
-            rel="noopener noreferrer"
+const Footer = () => {
+  return (
```

Comment:
This layout could be simplified by `grid`

Reply:
I will try to use gird